client/src/components/lecturesRowList/controller.js

```diff
@@ -0,0 +1,25 @@
+require('./style.scss');
+
+function LecturesRowListController($http, $state, $timeout) {
+    let self = this;
+
+    $http.get('/api/lectures').then(function(res) {
+        self.lectures = res.data;
+    });
+
+    $timeout(function () {
+        $('.collapsible').collapsible({
```

Comment:
Placing this into $postLink hook should avoid usage of $timeout function. You should not use $timeout and $interval function when its not absolutely necessary.